CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ All notable changes to this project will be documented in this file. This projec
 -   Restrict access to builds based on `ContentScopeModule`
 -   Decouple KubernetesModule from BuildsModule (Breaking: BuildsModuleConfig is now KubernetesModuleConfig)
 -   Breaking: AuthModule removes support for Bearer-Token
+-   Breaking: Drop support for 1.21 and prior
```

Comment:
"drop support": from https://github.com/kubernetes-client/javascript#compatibility: "Generally speaking newer clients will work with older Kubernetes, but compatability isn't 100% guaranteed." I think for those basic operations we make we won't have problems. Or do you know of any?

(and you should also mention that this is about kubernetes version...)

Reply:
No, I don't think we have any problems. But we should keep the client in sync with currently supported kubernetes versions.
> The Kubernetes project maintains release branches for the most recent three minor releases

from https://kubernetes.io/releases/